browser/ui/webui/new_tab_page/brave_new_tab_ui.cc

```diff
@@ -127,6 +135,10 @@ BraveNewTabUI::BraveNewTabUI(content::WebUI* web_ui, const std::string& name)
                                 std::make_unique<NTPCustomImagesSource>(
                                     ntp_custom_background_images_service));
   }
+
+  source->OverrideContentSecurityPolicy(
+      network::mojom::CSPDirectiveName::FrameSrc,
+      std::string("frame-src ") + kNTPSponsoredRichMediaUrl + ";");
```

Comment:
@aseren `base::StringPrintf`

Reply:
Maybe `base::StrCat`?

Reply:
Fixed with base::StringPrintf, unless there is a reason you prefer StrCat? Thanks

Reply:
No strong feelings :smile: